doc/going-further/helpers/notify.md

```diff
@@ -37,6 +37,66 @@ function notify()
 }
 ```
 
+## Choose when to display a notification
+
+You can set the `notify` property in the context to `false` to disable notifications globally:
+
+```php
+use Castor\Attribute\AsTask;
+
+use Castor\Context;
+use function Castor\notify;
+
+#[AsTask()]
+function notify()
+{
+    $context = new Context(notify: false);
+    notify('Hello world!'); // will not display a notification
```

Comment:
Why?, the default context (which is used here) should display notifications

Reply:
Yes, it lacks clarity, in fact `$context` is the context used for the call of `notify`.
I'll wrap it in the `with` method